doc/Makefile

```diff
@@ -46,7 +48,6 @@ clean:
 	$(RM) -r ../html ../info
 
 install:
-	$(if $(PREFIX),,$(error variable "PREFIX" must be set))
```

Comment:
Why do you remove these error statements? Even with your default, if the user sets an empty PREFIX, this errors are helpful, aren't they?

Reply:
That's essentially setting the wrong path. If someone clears PREFIX that's probably with intention and these error messages break the process – am guess that they'll never kick in anyhow.

Reply:
ACK you want it, you got it :-)

Reply:
Awesome! Saves quite some time typing. Thank you and stay healthy. :)